exporter/awsxrayexporter/go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	golang.org/x/net v0.0.0-20200625001655-4c5254603344
 	google.golang.org/grpc/examples v0.0.0-20200728194956-1c32b02682df // indirect
 )
+
+replace go.opentelemetry.io/collector v0.7.0 => ../../../opentelemetry-collector
```

Comment:
Is this for local testing of some sort? We already depend on the collector - can you revert all the go.mod changes?

Reply:
yes this is done for local testing, sure I will revert all the go.mod changes.